README.md

```diff
@@ -219,20 +216,24 @@ There may be cases where the _idem-api_ module is able to request the website's
+|----------|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------|-------------------|-------------------|
+| **0x00** | Australian driver licence    | A driver licence with your photo issued in your name. This includes physical and digital driver licences, learner permits and provisional licences.                  | Given name, middle, family name, dob, license number, residential address              | Primary           | Yes               |
+| **0x01** | Australian passport          | A passport issued by a country.                                                                                                                                      | Given name, middle name, family name, residential address, DOB, passport number        | Primary           | Yes               |
+| **0x02** | Australian birth certificate | A full birth certificate in your name or former name issued by Births, Deaths and Marriages. We can’t accept birth extracts or birth cards.                          | Given name, middle name, family name, DOB                                              | Primary           | No                |
+| **0x03** | Bank statement               | A bank statement issued in your name. Must be issued within the last 3 months.                                                                                       | Given name, middle name, family name, residential address                              | Secondary         | No                |
+| **0x04** | Rates notice                 | A paid rates notice issued in your name with your address that is less than 12 months old.                                                                           | Given name, middle name, family name, residential address                              | Secondary         | No                |
+| **0x05** | Utility account              | Water, gas, electricity or phone account with a receipt number. This must be in your name, show your address and be less than 12 months old. Must be a paid account. | Given name, middle name, family name, residential address                              | Secondary         | No                |
+| **0x06** | Medicare Card                | A current Medicare Card issued in your name.                                                                                                                         | Given name, middle name, family name, medicare number, expiry date, position in family | Secondary         | No                | 
```

Comment:
If you're including the medicare card's expiry date, then should you also include expiry date for drivers licence and passport?

Reply:
Yes good point, not needed right now in the idem app. Will remove.